auslib/web/public/client.py

```diff
@@ -68,7 +68,10 @@ def getCleanQueryFromURL(url):
         if len(force_split) < 2:
             force_split = force_value.split('%3F', 1)
 
-        query['force'] = int(force_split[0])
+        if force_split[0]:
```

Comment:
In the bug, I suggest that we should only ever set force to True if it is explicitly "1" - not for any other value. I think this will set it to True whenever it's not an empty string.